src/System.Private.CoreLib/shared/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Float.cs

```diff
@@ -90,22 +90,21 @@ public static bool TryFormat(float value, Span<byte> destination, out int bytesW
                     return FormattingHelpers.TryFormatThrowFormatException(out bytesWritten);
             }
 
-            string formatString = format.ToString();
-            string utf16Text = value.ToString(formatString, CultureInfo.InvariantCulture);
-            int length = utf16Text.Length;
-            if (length > destination.Length)
+            ReadOnlySpan<char> formatSpan = stackalloc char[] {format.Symbol};
```

Comment:
This doesn't look correct, though: the symbol won't include precision.  You can see all of the logic going into the ToString that was being used here: https://github.com/dotnet/coreclr/blob/356a08f7e90113b31005c02abb14662bc40bed70/src/System.Private.CoreLib/shared/System/Buffers/StandardFormat.cs#L126-L162.

Reply:
Yes I oversaw that and I guess that proves that the tests didn't work locally. I'll revert that to format.ToString()

Reply:
This precision code in your snippet above looks weird, I think it could use a bit of work?, i.e. Math.DivRem (or similar if that's not available). That (precision / 100) % 10);  is weird, precision is a byte, so why should it ever need % 10 after division by 100?

Isn't that what https://github.com/dotnet/coreclr/blob/356a08f7e90113b31005c02abb14662bc40bed70/src/System.Private.CoreLib/shared/System/Buffers/Text/Utf8Formatter/FormattingHelpers.cs#L162-L176 is supposed to do? (uint instead of byte)